### PR TITLE
issue-37 - updating regex for numbers

### PR DESCRIPTION
**Pull request by warwickandrew**

### CODE REVIEW

**Comment by moloko** [comment]
@warwickandrew this looks to have been based off an old version of this extension, the `wordCharacters` variable was removed as part of https://github.com/cgkineo/adapt-search/pull/28

@oliverfoster this does highlight that defining things like this as a massive string makes it really hard to see what's changed when it gets updated like this. Might be worth thinking about defining initially as an array then doing something like `wordChars = "\" + wordCharacters.join("\")`

**Comment by oliverfoster** [comment]
@warwickandrew could I have an issue or description of what you're trying to achieve here?

**Comment by moloko** [comment]
Perhaps related to #37 ?

File: js/search-algorithm.js [hunk 1]
```diff
@@ -114,22 +114,439 @@ define([
 
     };
 
-    var search = _.extend({
+        var wordCharacters = [
```
Comment (moloko) [inline js/search-algorithm.js]: 👍

File: js/search-algorithm.js [hunk 2]
```diff
@@ -114,22 +114,439 @@ define([
...
+            '\uFF66-\uFFBE',
+            '\uFFC2-\uFFC7',
+            '\uFFCA-\uFFCF',
+            '\uFFD2-\uFFD7',
+            '\uFFDA-\uFFDC'
+        ];
+
+        wordCharacters = wordCharacters.join('');
+		
+		var search = _.extend({
```
Comment (oliverfoster) [inline js/search-algorithm.js]: could you sort out the indentation here? it wasn't right to start with but we may as well correct it whilst you're here.

**Pull request merged**